server/src/models/Movie.js

```diff
@@ -38,6 +38,18 @@ class Movie extends Model {
                     from: "movies.id",
                     to: "reviews.movieId"
                 }
+            },
+            users: {
```

Comment:
is this association being used? Do you ever need to see ALL the Users that left Reviews for a Movie? I imagine there are queries to know the User that created a review - if this association isn't being used I would remove it. During core curriculum we encouraged you to define all associations, but now we don't want to create more work for ourselves, and should only add code that is being used